doc/classes/Dictionary.xml

```diff
@@ -4,16 +4,63 @@
 		Dictionary type.
 	</brief_description>
 	<description>
-		Dictionary type. Associative container which contains values referenced by unique keys. Dictionaries are always passed by reference.
+		Dictionary type. Associative container which contains values referenced by unique keys. Just like in real life each dictionary composes multiple pairs of unique keys and values. You can define a dictionary by placing a comma separated list of key:value pairs in curly braces [code]{}[/code].
+		[codeblock]
+		var my_dir = {
+		    key1: value1,
+		    key2: value2,
+		    key3: value3
+		}
+		[/codeblock]
 		Erasing elements while iterating over them [b]is not supported[/b].
 		Creating a dictionary:
```

Comment:
You already show how to create a dictionary with the example above, so this is redundant.

Reply:
I combined them into one would that be better?